### DISCUSSION TITLE
トップページで同一の方の記事の表示について

### DISCUSSION THREAD

**Discussion by simon-ritchie** [increments/qiita-discussions#244] [要望・アイデア・不具合]
たまーに発生しているのですが、トップページで同一の方の記事が連続して多く表示されることがあり、且つ記事のタグもフォローしていない（興味から外れた記事が続く）ケースがあるのでユーザーやトピックがばらけるようにしたりフォローしていないタグが連続しないようになると良いかもしれません！

![image](https://user-images.githubusercontent.com/25178045/175798673-ea308687-7ced-4e00-b929-75c450b99f69.png)

**Comment by tanaka-asami415** [comment]
@simon-ritchie 
ご意見ありがとうございます🙏
トップページは今後もアップデートを予定しておりますので
今回頂いた内容も含めて改善していきたいと思っております💪

**Comment by kato647** [comment]
@simon-ritchie 
大変お待たせいたしました :bow:
ホーム画面の記事の表示ロジックを改修しました。
ホーム画面をご確認いただければと思います！

**Reply by simon-ritchie** [reply]
@kato647 

おお、ご対応ありがとうございます・・・！🙇‍♂️
確認しましたが大丈夫そうです！

稀な発生ではあるので、将来万一再発したら別途共有させていただきます！
